src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -597,12 +626,27 @@ impl HummockVersion {
                         BTreeSet::from_iter(group_construct.table_ids.clone())
                     };
 
-                self.init_with_parent_group(
-                    parent_group_id,
-                    *compaction_group_id,
-                    member_table_ids,
-                    group_construct.get_new_sst_start_id(),
-                );
+                if group_construct.version >= CompatibilityVersion::SplitGroupByTableId as _ {
+                    let split_key = if group_construct.split_key.is_some() {
+                        Some(Bytes::from(group_construct.split_key.clone().unwrap()))
+                    } else {
+                        None
```

Comment:
Is this branch reachable?

Reply:
https://github.com/risingwavelabs/risingwave/blob/fd8f4b013c03542918444272dbd2fec4bb58f5f1/src/meta/src/hummock/manager/transaction.rs#L153

reachable when construct a new compaction group